docs/OnlineTutorial/guide.md

```diff
@@ -807,8 +807,8 @@ function max(a: int, b: int): int
 {
   // Use your code from Exercise 4
 }
-method Testing() {
-  // Add assertions to check max here. Be sure to capture it to a local variable
+method Testing() returns (r: int) {
+  // Add assertions to check max here. Be sure to capture it to the out parameter
```

Comment:
```suggestion
  // Add assertions to check max here. Be sure to capture it to the out-parameter
```
Also, there is one more occurrence of "out parameter" in the doc in section 13.3, can you also fix it?
There are 53 occurrences of "out-parameter" (with the dash) in the documentation, so I assume it's the correct one.

Reply:
Fixed the other occurrence as well.